### PR TITLE
Add stratos assets

### PR DESCRIPTION
**Pull request by BoThe1K**
### Adding Assets

If adding a new asset, please ensure the following:
- [x] Asset is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
- [x] Add asset to bottom of `zone_assets.json`.
   - [x] `chain_name` and `base_denom` are provided and use values exactly as defined at the Chain Registry.
   - [x] `path` is provided, and the IBC channel referenced is registered at the Chain Registry (unless native to Osmosis).
   - [x] `osmosis_verified` is set to `false`
   - [x] `listing_date_time_utc` is specified and accurate
   - [x] Optional: `transfer_methods`, `peg_mechanism`, `override_properties`, `canonical`, `categories`, where necessary (see [README](https://github.com/osmosis-labs/assetlists/tree/main?tab=readme-ov-file#how-to-add-assets) for details).
- [x] I am aware that upgrading an asset to 'verified' status requires an additional PR to this repo (checklist below).  

### Adding Chains

If adding a new chain, please ensure the following:
- [x] Chain is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
   - Chain's registration must have `staking` defined, with at least one `staking_token` denom specified.
   - Chain's registration must have `fees` defined; at least one fee token has low, average, and high gas prices defined.
- [x] IBC Connection between chain and Osmosis is registered.
- [x] Add chain to bottom of `zone_chains.json`
   - [x] `rpc` and `rest` does not have any CORS blocking of the Osmosis domain, and RPC node has have WSS enabled.
   - [x] `explorer_tx_url` correctly directs to the transaction when the hash is inserted into the URL.

### CODE REVIEW

File: osmosis-1/osmosis.zone_assets.json [hunk 1]
```diff
@@ -5249,6 +5249,16 @@
       "osmosis_verified": false,
       "listing_date_time_utc": "2024-09-26T03:00:00Z",
       "_comment": "SinGarden token"
+    },
+    {
+      "chain_name": "stratos",
+      "base_denom": "wei",
+      "path": "transfer/channel-81016/wei",
+      "categories": ["chain"],
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:5257]: Please remove this category

Comment (BoThe1K) [reply osmosis-1/osmosis.zone_assets.json]: @JeremyParish69 removed

**Comment by JeremyParish69** [comment]
The validation failed because there aren't any fee rates defined at the Chain Registry

**Comment by BoThe1K** [comment]
> The validation failed because there aren't any fee rates defined at the Chain Registry https://github.com/cosmos/chain-registry/blob/master/stratos/chain.json#L18-L24

thanks, have created PR: https://github.com/cosmos/chain-registry/pull/5415

anything else we should add here or in chain-registry?

**Comment by JeremyParish69** [comment]
Oh, the IBC connection between osmosis and stratos must be registered. There is a checklist item for this requirement.
https://github.com/cosmos/chain-registry/tree/master/_IBC

**Comment by BoThe1K** [comment]
> Oh, the IBC connection between osmosis and stratos must be registered. There is a checklist item for this requirement. https://github.com/cosmos/chain-registry/tree/master/_IBC

thanks, have created PR: https://github.com/cosmos/chain-registry/pull/5418

**Comment by JeremyParish69** [comment]
Looks like the check passed. I think the last required change is to resolve conflicts.

**Comment by BoThe1K** [comment]
@JeremyParish69 I have a question. When we could see our asset on https://app.osmosis.zone/ (even as unverified through the settings filter)? (so when it will become available)

**Comment by JeremyParish69** [comment]
> @JeremyParish69 I have a question. When we could see our asset on https://app.osmosis.zone/ (even as unverified through the settings filter)? (so when it will become available)

Will need to submit to Osmosis Labs' assetlist https://github.com/osmosis-labs/assetlists?tab=readme-ov-file#how-to-add-assets

**Comment by BoThe1K** [comment]
Hi @JeremyParish69 ! Even after https://github.com/osmosis-labs/assetlists/pull/1940 , I do not see STOS in the pool creation or the swap (with unverified asset checkbox in the settings). Should we add something else for this?

Screenshot 2024-10-07 at 15 49 55
Screenshot 2024-10-07 at 15 51 43

**Comment by JeremyParish69** [comment]
> Hi @JeremyParish69 ! Even after #1940 , I do not see STOS in the pool creation or the swap (with unverified asset checkbox in the settings). Should we add something else for this?

No further action required. Updates to Osmosis Zone are delayed, but should be recovered soon.

**Pull request merged**